readme.md

```diff
@@ -22,6 +21,12 @@ a roots extension for inlining your CSS properties into the style attribute in a
     extensions: [ rootsInlineCss(options) ]
   ```
 
+### Usage
+
+  This extension provides a way for you to inline your html with css. By default all of your `html` files will be inlined however you can specify through options which files you want inlined either with a single file or a [glob expression](http://www.globtester.com/).
```

Comment:
i'd rather we reference the actual https://github.com/isaacs/minimatch library, since it will provide the dev with api reference